python/lsst/pipe/tasks/postprocess.py

```diff
@@ -798,10 +799,31 @@ class TransformObjectCatalogConfig(TransformCatalogBaseConfig,
         doc=("Whether results dataframe should have a multilevel column index (True) or be flat "
              "and name-munged (False).")
     )
+    goodFlags = pexConfig.ListField(
+        dtype=str,
+        default=[],
+        doc=("List of 'good' flags that should be set False when filling empty tables. "
+             "All other flags are considered to be 'bad' flags and will be set to True.")
+    )
+    floatFillValue = pexConfig.Field(
+        dtype=float,
+        default=np.nan,
+        doc="Fill value for float fields when filling empty tables."
+    )
+    integerFillValue = pexConfig.Field(
+        dtype=int,
+        default=-1,
+        doc="Fill value for integer fields when filling empty tables."
+    )
 
     def setDefaults(self):
         super().setDefaults()
         self.primaryKey = 'objectId'
+        self.goodFlags = ['calib_astrometry_used',
+                          'calib_photometry_reserved',
+                          'calib_photometry_used',
+                          'calib_psf_candidate',
+                          'calib_psf_used']
```

Comment:
We are missing `calib_psf_reserved` from this list:
https://github.com/lsst/obs_subaru/blob/master/policy/Object.yaml#L641